applications/tests/test_applications_new_schema_queries.py

```diff
@@ -22,6 +23,7 @@
 def test_berth_applications_no_customer_filter_true(
     berth_application, superuser_api_client
 ):
+    berth_application.lease = BerthLeaseFactory()
```

Comment:
why did you have to add lease to all of these applications? it's allowed to be null 🤔

Reply:
Because we're adding "invalid" statuses to Applications without leaes, and they fail the validation
```python
 if (
    not hasattr(self, "lease")
    and self.status not in allowed_statuses_without_lease
 ):
```

Reply:
but the default application status is `PENDING` and it allows `application.lease` to be `None` 🤔

Reply:
Ahhh, good catch. The first two (`test_berth_applications_no_customer_filter_true` and `test_berth_applications_no_customer_filter_false`) don't require it. I got too excited with all the copy-pasting going around.